doc/api/errors.md

```diff
@@ -563,6 +570,53 @@ found [here][online].
   encountered by [`http`][] or [`net`][] -- often a sign that a `socket.end()`
   was not properly called.
 
+<a id="nodejs-error-codes"></a>
+## Node.js Error Codes
+
+<a id="ERR_INVALID_ARG_TYPE"></a>
+### ERR_INVALID_ARG_TYPE
+
+The `'ERR_INVALID_ARG_TYPE'` error code is used generically to identify that
```

Comment:
(Not sure which PR is the right place to ask, this one seems the oldest so I'll ask here)

Which one does the "type" here cover:

* Primitive types (i.e. `typeof` types, doesn't differentiate `Date` and `RegExp` and stuff)
* Builtin type tags(i.e. results of `Object.prototype.toString.call()`, differentiates `Date` and `RegExp`)
* Class mismatch(i.e. `instanceof` types, differentiates `Date` and `RegExp`)